forest/benchmarking/plotting/state_process.py

```diff
@@ -110,11 +110,12 @@ def plot_pauli_transfer_matrix(ptransfermatrix, ax, labels=None, title='', fonts
     ticklabs = cb.ax.get_yticklabels()
     cb.ax.set_yticklabels(ticklabs, ha='right')
     cb.ax.yaxis.set_tick_params(pad=35)
+    cb.draw_all()
     ax.set_xticks(range(dim_squared))
     ax.set_xlabel("Input Pauli Operator", fontsize=fontsizes)
     ax.set_yticks(range(dim_squared))
     ax.set_ylabel("Output Pauli Operator", fontsize=fontsizes)
-    ax.set_title(title, fontsize= int(np.floor(1.2*fontsizes)))
+    ax.set_title(title, fontsize= int(np.floor(1.2*fontsizes)), pad=15)
```

Comment:
This pad causes an `AttributeError: Unkown property pad`  for me when running random_operators.ipynb

Reply:
This is probably a matplotlib version problem. Perhaps we should upgrade the requirements.